test/validation/api/scheduler/scheduler.c

```diff
@@ -2348,6 +2347,130 @@ static void scheduler_test_order_wait_1_thread(void)
 	CU_ASSERT(odp_queue_destroy(queue) == 0);
 }
 
+static int order_wait_helper(void *arg ODP_UNUSED)
+{
+	odp_event_t ev;
+
+	ev = odp_schedule(NULL, ODP_SCHED_WAIT);
+	if (ev == ODP_EVENT_INVALID) {
```

Comment:
Some wait time would be needed (e.g. 10 sec) to avoid thread getting stuck (in CI) when something goes wrong. ODP_SCHED_WAIT option should not ever return EVENT_INVALID, but wait until there is an event.